cloudinit/config/cc_apt_configure.py

```diff
@@ -575,8 +575,9 @@ def is_deb822_sources_format(apt_src_content: str) -> bool:
 
 
 DEFAULT_APT_CFG = {
-    "sourcelist": "/etc/apt/sources.list",
-    "sourceparts": "/etc/apt/sources.list.d/",
+    "Dir::Etc": "etc/apt",
```

Comment:
Refactoring code that hasn't landed yet adds churn / noise in the commit log. To avoid this, please squash the changes in this commit into the commit that introduced these lines of code prior to merging.

Reply:
yes agreed on this point, which had prompted my overall squash merge for this PR given some of the overlap and churn I had introduced when handling review comments and iterations. I left these separate commits in a less than desirable state after handling initial reviews which made separate commits for this effort a bit more work than I wanted to spend on this particular PR.